src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelectorAll('li');
+
+function sortList(lists) {
```

Comment:
```suggestion
function sortList(elements) {
```
ты же получаешь набор элементов, а не набор наборов элементов